.gitignore

```diff
@@ -39,4 +39,4 @@ vignettes/*.pdf
 # R Environment Variables
 .Renviron
 inst/doc
-docs
+#docs
```

Comment:
I'm not sure if this is the right way to do it. I don't think we want files from docs being committed into the main branch, and therefore should ignore them since they will be created in the gh-pages branch automatically once PRs are merged. What do you think?